.github/workflows/run-tests.yml

```
@@ -4,6 +4,8 @@ run-name: "Run Tests: ${{ github.ref_type }} ${{ github.ref_name }} by ${{ githu
 on:
   workflow_dispatch:
   workflow_call:
+  pull_request:
+    branches: [ rewrite ]
```

YAMLlint (1.37.1)

[error] too many spaces inside brackets

(brackets)